documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md

```diff
@@ -0,0 +1,31 @@
+## Description
+
+This module allows you to authenticate Advantech WebAccess.
+
+## Vulnerable Application
+
+This module was specifically tested on versions 8.0, 8.1, and 8.2:
+
+**8.2 Download**
+
+http://advcloudfiles.advantech.com/web/Download/webaccess/8.0/AdvantechWebAccessUSANode8.0_20141103_3.4.3.exe
+
+**8.1 Download**
+
+http://advcloudfiles.advantech.com/web/Download/webaccess/8.1/AdvantechWebAccessUSANode8.1_20151230.exe
+
+**8.0 Download**
+
+http://advcloudfiles.advantech.com/web/Download/webaccess/8.0/AdvantechWebAccessUSANode8.0_20141103_3.4.3.exe
+
+
```

Comment:
Can you either add a comment here, or another heading for Variables, or Defaults where it mentions admin:'' is the default creds?  Just wanna get that captured in the docs as well (even though its in the module description)